manifests/service.pp

```diff
@@ -3,8 +3,8 @@
   exec { 'pulp refresh system service':
     command     => '/bin/systemctl daemon-reload',
     refreshonly => true,
-  } ->
-  service { ['pulp_celerybeat', 'pulp_workers', 'pulp_resource_manager', 'pulp_streamer']:
+  }
+  -> service { ['pulp_celerybeat', 'pulp_workers', 'pulp_resource_manager', 'pulp_streamer']:
```

Comment:
@ekohl I agree this looks particularly bad.  All the other changes look fine either way to me.